src/main/java/ai/elimu/web/servlet/CustomDispatcherServlet.java

```diff
@@ -60,11 +62,12 @@ protected WebApplicationContext initWebApplicationContext() {
      */
     private void createJpaSchemaExport() {
         logger.info("createJpaSchemaExport");
-        
+
         ServiceRegistry serviceRegistry = new StandardServiceRegistryBuilder()
 //                .configure("META-INF/jpa-persistence.xml")
                 .applySetting("hibernate.dialect", ConfigHelper.getProperty("jpa.databasePlatform"))
                 .applySetting("hibernate.hbm2ddl.auto", "update")
+                .applySetting(AvailableSettings.CONNECTION_PROVIDER, new ConnectionProviderWeb(ConfigHelper.getProperty("jdbc.url"), ConfigHelper.getProperty("jdbc.username"), ConfigHelper.getProperty("jdbc.password")))
```

Comment:
I suggest preparing a separate variable for improved readability:
```suggestion
        ConnectionProviderWeb connectionProviderWeb = new ConnectionProviderWeb(ConfigHelper.getProperty("jdbc.url"), ConfigHelper.getProperty("jdbc.username"), ConfigHelper.getProperty("jdbc.password"));
        ServiceRegistry serviceRegistry = new StandardServiceRegistryBuilder()
//                .configure("META-INF/jpa-persistence.xml")
                .applySetting("hibernate.dialect", ConfigHelper.getProperty("jpa.databasePlatform"))
                .applySetting("hibernate.hbm2ddl.auto", "update")
                .applySetting(AvailableSettings.CONNECTION_PROVIDER, connectionProviderWeb)
```